gpt_playground/settings.py

```diff
@@ -506,3 +508,6 @@
 
 # Django Field Audit
 FIELD_AUDIT_AUDITORS = ["apps.audit.auditors.RequestAuditor", "field_audit.auditors.SystemUserAuditor"]
+
+# tz_detect
+TZ_DETECT_COUNTRIES = ["ZA"]
```

Comment:
I think we want to include more than just ZA since this list is checked prior to the full list. My suggestion would be:

```suggestion
TZ_DETECT_COUNTRIES = ["US", "IN", "GB", "ZA", "KE"]
```

(Though looking at the [code](https://github.com/adamcharnock/django-tz-detect/blob/develop/tz_detect/utils.py#L36-L38) I'm not sure it really makes a difference)